pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -227,43 +235,29 @@ func (db *DB) locked() func() {
 	return db.mu.Unlock
 }
 
-// DeleteExpired deletes expired pieces
-func (db *DB) DeleteExpired(ctx context.Context) (expired []string, err error) {
-	defer mon.Task()(&ctx)(&err)
-	defer db.locked()()
-
-	// TODO: add limit
-
-	tx, err := db.db.BeginTx(ctx, nil)
-	if err != nil {
-		return nil, err
+// DeleteObsolete deletes obsolete pieces
+func (db *DB) DeleteObsolete(path string) (err error) {
+	if path == "" {
+		zap.S().Warnf("Empty path: %+v", path)
```

Comment:
Don't use global logger `zap.S()`

Reply:
Rest of the code in the psdb.go file uses same Zap.S(). Just to be consistent I used the same. what do you recommend to use?

Reply:
Don't log this in method. In migration step throw error if path is empty. It shouldn't happen.